benchmark/index.js

```diff
@@ -42,16 +44,44 @@ if (!skipCreate) {
   })
 }
 
-test('initial indexing', (t) => {
+test('reset db2', (t) => {
   rimraf.sync(db2Path)
+  t.pass('delete db2 folder to start clean')
+  t.end()
+})
 
+test('migration', (t) => {
   const keys = ssbKeys.loadOrCreateSync(path.join(dir, 'secret'))
   const sbot = SecretStack({ appKey: caps.shs })
-    .use(require('../'))
-    .call(null, {
-      keys,
-      path: dir,
+    .use(require('../migrate'))
+    .call(null, { keys, path: dir })
+
+  const start = Date.now()
+  sbot.db2migrate.start()
+
+  pull(
+    fromEvent('ssb:db2:migrate:progress', sbot),
+    pull.filter((progress) => progress === 1),
+    pull.take(1),
+    pull.drain(() => {
+      const duration = Date.now() - start
+      t.pass(`duration: ${duration}ms`)
+      fs.appendFileSync(reportPath, `- Migration: ${duration}ms\n`)
+      // wait for new log FS writes to finalize
```

Comment:
Shouldn't a onDrain work here?

Reply:
Oh, going to try that.

Reply:
Oh, the problem here is that I don't have `sbot.db`, so I don't have access to the `log` :(

Reply:
Ah right, okay.

Reply:
Feel free to merge :)